nginx.conf.template

```diff
@@ -25,6 +25,13 @@ http {
       proxy_set_header        X-Real-IP $remote_addr;
       proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
       proxy_set_header        X-Forwarded-Proto $scheme;
+
+      # enables WS support
+      proxy_http_version 1.1;
+      proxy_set_header Upgrade $http_upgrade;
+      proxy_set_header Connection "upgrade";
+      proxy_read_timeout 86400s
+      proxy_send_timeout 86400s
```

Comment:
Adding `websocket_connect_timeout 1d` could be helpful for development purpose, how do you think?